imagery/i.cluster/i.cluster.md

```diff
@@ -60,26 +57,26 @@ classify only the same imagery group used for generating signatures.
 
 ### Parameters
 
-**group=***name*  
+**group:**
```

Comment:
Should these parameters be in the same line as the text (as is currently rendered on GitHub)?
(Needing a `<br />` or double trailing whitespace)

Also, was removing the type of parameter for uniformity with other files?
For sample, having it showing "rows,cols" seems useful.
If it was because of broken italics, there's underscores for mixing with bold.

```suggestion
**group:**_name_  
```